docs/admin/install/slack_setup.md

```diff
@@ -28,24 +30,221 @@ PLUGINS_CONFIG = {
 }
 ```
 
+!!! info
+    When running Slack in Socket Mode, Nautobot ChatOps Plugin can specify where the static files are hosted.
+    This is helpful for giving Slack access to only the server hosting Nautobot Static Files. If Django Storage
+    is configured, Nautobot ChatOps Plugin will use the static host from Django Storage. This is optional as
+    ChatOps only uses static files for sending logos with messages. To configure the static files host, use
+    `slack_socket_static_host` ex. `slack_socket_static_host: 'https://example.com/static/'`.
+
+## Connecting to Slack
+
+The Nautobot ChatOps plugin supports two methods of communicating with Slack.
+
+1. Incoming Webhooks
+2. Socket Mode
+
+The original supported method of connection was only incoming webhooks. As of version 1.10, socket mode support was added for Slack.
+
+### Webhooks vs. Socket Mode
+
+With incoming webhooks, whenever a user sends a message to the bot via the client application (phone or desktop app), (1) the connection is initiated from the client, and the message gets sent to Slacks cloud. The Slack servers then initiate a connection to the ChatOps plugin running on the Nautobot server (2). Since this is a new, incoming connection, it must be port forwarded and allowed through any firewalls in your network between the internet and your Nautobot server (3). Nautobot then replies to the existing connection to Slack, which in turn forwards the response to the client.
```

Comment:
```suggestion
With incoming webhooks, whenever a user sends a message to the bot via the client application (phone or desktop app), (1) the connection is initiated from the client, and the message gets sent to Slack's cloud. The Slack servers then initiate a connection to the ChatOps plugin running on the Nautobot server (2). Since this is a new, incoming connection, it must be port forwarded and allowed through any firewalls in your network between the internet and your Nautobot server (3). Nautobot then replies to the existing connection to Slack, which in turn forwards the response to the client.
```